redaxo/src/addons/backup/pages/export.php

```diff
@@ -43,6 +43,7 @@
     // ------------------------------ FUNC EXPORT
 
     $exportfilename = strtolower($exportfilename);
+    /** @psalm-taint-escape shell */
     $filename = preg_replace('@[^\.a-z0-9_\-]@', '', $exportfilename);
```

Comment:
wirklich nur shell? Ich meine, ich hätte gestern gestern in den Fehlern gesehen, dass die Variable am Ende auch in fopen landet. Und das sollte hierdurch ja eigentlich auch sicher sein, oder?

Reply:
ich hab hier das "escaped" markiert, was psalm gemeckert hat.
ggf. hat sich das auch mit dem neuen release geändert